packages/accordions/demo/stepper.stories.mdx

```diff
@@ -22,7 +22,16 @@ import { STEPPER_STEPS as STEPS } from './stories/data';
 <Canvas>
   <Story
     name="Stepper"
-    args={{ hasIcon: false, isLabelHidden: false, steps: STEPS }}
+    args={{
+      hasIcon: false,
+      isLabelHidden: false,
+      steps: STEPS,
+      iconProps: {
+        role: 'img',
+        'aria-label': 'checked',
+        'aria-hidden': undefined
+      }
```

Comment:
We should see this under the "Stepper.Label" section of controls. Please add with the appropriate category to `argTypes` below.